docs/run.md

```diff
@@ -11,6 +11,16 @@ Build citrea:
 make build
 ```
 
+### Prequisites
+
+In order for the sequencer node to run, you need to have Postgres up and running.
```

Comment:
this is not needed - if no db_config is given in the sequencer_config.toml, then the sequencer doesnt need postgres

Reply:
I think we can just skip db stuff in readmes, only we are gonna use that feature of the sequencer